cmd/admin-router.go

```diff
@@ -161,6 +161,7 @@ func registerAdminRouter(router *mux.Router, enableConfigOps, enableIAMOps bool)
 	// -- KMS APIs --
 	//
 	adminRouter.Methods(http.MethodGet).Path(adminAPIVersionPrefix + "/kms/key/status").HandlerFunc(httpTraceAll(adminAPI.KMSKeyStatusHandler))
+	adminRouter.Methods(http.MethodPost).Path(adminAPIVersionPrefix + "/kms/key/rotate").HandlerFunc(httpTraceAll(adminAPI.KMSKeyRotateHandler))
```

Comment:
it would be better to just trace the headers for this API - since the response body can be large.

Reply:
Yes, good catch :+1: